Tasks/IISWebAppManagementOnMachineGroup/Tests/L0AppcmdAddUpdateAppPool.ps1

```diff
@@ -0,0 +1,62 @@
+[CmdletBinding()]
+param()
+
+. $PSScriptRoot\..\..\..\Tests\lib\Initialize-Test.ps1
+. $PSScriptRoot\MockHelpers.ps1
+
+. $PSScriptRoot\..\ps_modules\TaskModuleIISManageUtility\AppCmdOnTargetMachines.ps1
+
+$appPoolName = "Sample App Pool"
+$dotNetVersion = "v4.0"
+$pipeLineMode = "Integrated"
+$appPoolIdentity = "ApplicationPoolIdentity"
+$appPoolCredentials = $null
+
```

Comment:
These tests should be vsts-rm-extensions repro. Here we should add only tests related to utility.ps1 file.